misc/userscripts/qute-pass

```diff
@@ -108,6 +108,10 @@ class ExitCodes(enum.IntEnum):
     COULD_NOT_MATCH_USERNAME = 3
     COULD_NOT_MATCH_PASSWORD = 4
 
+class CouldNotMatchUsername(Exception):
+    pass
+class CouldNotMatchPassword(Exception):
+    pass
```

Comment:
I believe userscripts are not checked for PEP8 styling so you get no warning here, but ideally you should surround top-level function and class definitions with two blank lines.
```suggestion


class CouldNotMatchUsername(Exception):
    pass
    
    
class CouldNotMatchPassword(Exception):
    pass
    
```